### ISSUE TITLE
background not loading when having spaces un name

### ISSUE THREAD

**Issue by jbdemonte** [hakimel/reveal.js#1834]
Background seems to require to be renamed without spaces when used in background

This works:
```html
    <section data-background-image="assets/a-new-hope.jpg"></section>
```
While this doesn't:
```html
    <section data-background-image="assets/a new hope.jpg"></section>
```

**Comment by RobertBaron** [comment]
I've open a pull request for this
https://github.com/hakimel/reveal.js/pull/1836

If the url has any special characters, they should be escaped, or in the case of white-space, at least quoted.
See [w3c spec](https://www.w3.org/TR/CSS2/syndata.html#value-def-uri) for full requirements

**Comment by bnjmnt4n** [comment]
Closed by #1836.
